libraries/apollo-normalized-cache-api-incubating/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/api/internal/RecordWeigher.kt

```diff
@@ -23,9 +22,9 @@ internal object RecordWeigher {
 
   @JvmStatic
   fun calculateBytes(record: Record): Int {
-    var size = SIZE_OF_RECORD_OVERHEAD + record.key.utf8Size().toInt()
+    var size = SIZE_OF_RECORD_OVERHEAD + record.key.length
     for ((key, value) in record.fields) {
-      size += key.utf8Size().toInt() + weighField(value)
+      size += key.length + weighField(value)
```

Comment:
What's the reasoning for changing the weighter implementation?

Reply:
Yes sorry this is a totally unrelated to this PR! I just noticed this and thought in memory strings are stored either in utf-16 or some sort of compact form that's iso-latin-1 if possible (apparently [since JDK9](https://www.baeldung.com/java-9-compact-string) - I can't tell if this applies to Android). I think it's never utf-8. But `length*2` may be a better value if we think it's probably utf-16. 

And now the fun question: what is it on apple/js/wasm? 😅